neqo-transport/src/ecn.rs

```diff
@@ -38,73 +27,34 @@ enum EcnValidationState {
     Capable,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub struct EcnInfo {
     /// The current state of ECN validation on this path.
     state: EcnValidationState,
-
+    /// The number of packets sent so far on the path during the ECN validation.
+    sent: usize,
```

Comment:
This is not a great name if the number is only the number of packets that were sent during testing.  `test_packet_count` might work.